modules/features/dgu_print/dgu_print.module

```diff
@@ -0,0 +1,112 @@
+<?php
+/**
+ * @file
+ * Drupal needs this blank file.
+ */
+
+/**
+ * Implements hook_menu().
+ */
+function dgu_print_menu() {
+  $items['print/header'] = array(
+    'page callback' => 'dgu_print_header',
+    'access callback' => TRUE,
+  );
+  $items['print/footer'] = array(
+    'page callback' => 'dgu_print_footer',
+    'access callback' => TRUE,
+  );
+  return $items;
+}
+
+function dgu_print_header() {
+//[page] => 3
+//[section] => ISB Standard: Business Data Standards
+//[sitepage] => 3
+//[title] => Contributing qualification element (QE) outcome data standard
+//[subsection] => Standard identifier:ISB-000003
+//[frompage] => 1
+//[subsubsection] =>
+//[topage] => 3
+//[doctitle] => Contributing qualification element (QE) outcome data standard
+//[sitepages] => 3
+//[webpage] => -
+//[time] => 14:56:43
+//[date] => 22 Sep 2014
+
+  $page = (int)$_GET['page'];
+  $title = $_GET['title'];
+  $margin_top = strlen($title) > 90 ? 0 : 30;
+
+  global $theme_key;
+  $logo_path = theme_get_setting('logo', $theme_key);
+
+
+  if ($page > 1) {
+    print '<!DOCTYPE html><html>
+   <body style="position: relative;">
+     <div style="display: block; width: 100%; height: 55px; position: absolute; top: 0; left: 0; border-bottom: 1px solid #000;">
+     <img style="width:152px; height: 45px;" src="' . $logo_path . '" />
+     <div style="position: absolute; top: ' . $margin_top .'px; right: 0; width: 80%; text-align: right;"><strong>' . $title . '</strong></div>
+     </div>
+   </body></html>';
+    //syslog (LOG_DEBUG, 'called ' . print_r($title, 1));
+  }
+  else {
+    print '<!DOCTYPE html><html><body><br/></body></html>';
+    //syslog (LOG_DEBUG, 'called, printed empty ');
+  }
+
+}
+
+function dgu_print_footer() {
+//[page] => 3
+//[section] => ISB Standard: Business Data Standards
+//[sitepage] => 3
+//[title] => Contributing qualification element (QE) outcome data standard
+//[subsection] => Standard identifier:ISB-000003
+//[frompage] => 1
+//[subsubsection] =>
+//[topage] => 3
+//[doctitle] => Contributing qualification element (QE) outcome data standard
+//[sitepages] => 3
+//[webpage] => -
+//[time] => 14:56:43
+//[date] => 22 Sep 2014
+
+  syslog (LOG_DEBUG, 'called ' . print_r($_GET, 1));
+
+  $page = $_GET['page'];
+  $topage = $_GET['topage'];
+
+//  $id = $_GET['standard_identifier'];
+//  $type = $_GET['standard_type'];
+  $status = $_GET['standard_status'];
```

Comment:
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA1

I expect to remove them in a later commit before this module is enabled.
Pawel Ratajczak wrote on 08/01/2015 11:54:
>> +//[subsubsection] => +//[topage] => 3 +//[doctitle] =>
>> Contributing qualification element (QE) outcome data standard 
>> +//[sitepages] => 3 +//[webpage] => - +//[time] => 14:56:43 
>> +//[date] => 22 Sep 2014 + +  syslog (LOG_DEBUG, 'called ' .
>> print_r($_GET, 1)); + +  $page = $_GET['page']; +  $topage =
>> $_GET['topage']; + +//  $id = $_GET['standard_identifier']; +//
>> $type = $_GET['standard_type']; +  $status =
>> $_GET['standard_status'];
> 
> these are ISB variables
> 
> --- Reply to this email directly or view it on GitHub: 
> https://github.com/datagovuk/dgu_d7/pull/445/files#r22647622
> 

-----BEGIN PGP SIGNATURE-----
Version: GnuPG/MacGPG2 v2.0.22 (Darwin)
Comment: GPGTools - http://gpgtools.org

iQEcBAEBAgAGBQJUrnNxAAoJEFFVsRfbzGZ6U4YH/jtCgbyeJnNvc28coKmJj0nh
IKA7tOC+sdtYl0L+JhKkkgKH08WFh6bZ+XIYCUtlm4aqbdS/9FPwL8bI0Mbri1ks
POWJWb9WjkIafrhpAjwc0uh8f1cXypWx6OsuSMSzv0N8PJyAAjrSn2cS7hvE9kvj
hYH6e1VG8lzo/V7S/V/Aw7diNi/szg61RJWvoRG781M30xpgS57oaf3pqv+l/cdg
+fg+kEAU/Pw1bkP3AJODWSOe3W/Z8Ui00dmGSV+o+9yK7cK1J556vrG2BtyJ6NMR
qmCxmGvU4kOREeg4yfk0yt72XzdKOj9lVI57cy6iUBVOhSl21OvHFI+NxpSafWg=
=mQow
-----END PGP SIGNATURE-----